docs/content/using/csi_ug_using_sample.md

```diff
@@ -2,38 +2,26 @@
 
 **Note:** The secret names given can be user specified. When giving secret names when managing different system storage types, be sure to give system type indicators to each name.
 
-The following are examples of different types of secret names that can be given per storage type.
-
-|Storage system name|Secret name|
-|-------------------|-----------|
-|IBM FlashSystem® A9000 <br />IBM FlashSystem A9000R|a9000-array1|
-|IBM Spectrum Virtualize Family including IBM SAN Volume Controller and <br />IBM FlashSystem family members built with IBM Spectrum Virtualize <br />(including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)|storwize-array1|
-|IBM DS8000® Family products|DS8000-array1|
-
-**Note:** This procedure is applicable for both Kubernetes and Red Hat® OpenShift®. For Red Hat OpenShift, replace `kubectl` with `oc` in all relevant commands.
-
 Use this information to run a stateful container on StatefulSet volumes using either file systems or raw block volumes.
 
-1.  Create an array secret, as described in [Creating a Secret](../configuration/csi_ug_config_create_secret.md).
+1. Create an array secret, as described in [Creating a Secret](../configuration/csi_ug_config_create_secret.md).
 
-2.  Create a storage class, as described in [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
+2. Create a storage class, as described in [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
 
-    **Remember:** The `SpaceEfficiency` values for Spectrum Virtualize Family are: thick, thin, compressed, or deduplicated. These values are not case specific.
+    **Remember:** The `SpaceEfficiency` values for Spectrum Virtualize Family are: `thick`, `thin`, `compressed`, or `deduplicated`. These values are not case specific.
     
-    For DS8000 Family systems, the default value is standard, but can be set to thin, if required. These values are not case specific. For more information, see [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
+    For DS8000 Family systems, the default value is `none`, but can be set to `thin`, if required. These values are not case specific. For more information, see [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
     
     This parameter is not applicable for IBM FlashSystem A9000 and A9000R systems. These systems always include deduplication and compression.
 
-3.  Create a PVC with the size of 1 Gb, as described in [Creating a PersistentVolumeClaim (PVC)](../configuration/csi_ug_config_create_pvc.md).
+3. Create a PVC with the size of 1 Gb, as described in [Creating a PersistentVolumeClaim (PVC)](../configuration/csi_ug_config_create_pvc.md).
 
-4.  Display the existing PVC and the created persistent volume (PV).
+4. Display the existing PVC and the created persistent volume (PV).
```

Comment:
this step is not necessary for "running stateful containers with a storage volume provisioned from IBM® block storage systems."

Reply:
no but it's part of the user workflow. leaving as-is for now.

Reply:
could we at least make this step optional?

Reply:
DONE